src/index.html

```diff
@@ -1,13 +1,36 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="Styles/reset.css">
+    <link rel="stylesheet" href="Styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <div class="container__foto foto"></div>
+      <div class="container__text text">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+      <div class="container__code code">Код товара: 195434</div>
+      <div class="container__rate rate">
+        <div class="rate__stars--4 stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rate__review review">Отзывов: 5</div>
+      </div>
+      <div class="container__price price">
+        <div class="price__text">Цена:</div>
+        <div class="price__value">69 999 грн</div>
+      </div>
+      <button class="container__buy buy" data-qa="hover">КУПИТЬ</button>
```

Comment:
```suggestion
      <button class="container__buy buy" data-qa="hover">
        Купить
      </button>
```
используй text-transform: uppercase